plugins/notifications-backend/package.json

```diff
@@ -55,6 +55,7 @@
     "supertest": "6.3.3"
   },
   "files": [
-    "dist"
+    "dist",
+    "src/openapi.yaml"
```

Comment:
what is the use of adding the yaml here?

Reply:
So it becomes part of the tgz published to the NPM registry. It was missing and so failing build.

Reply:
how do the rest of the files get into the tgz? if they get to the tgz then so should this one

Reply:
only dist is javascript, correct? so that's what goes into the package? and u want to add the openapi spec. correct?

Reply:
We might place it into `dist` as well if it makes sense. The point is to have the file reachable when the package is added as an NPM dependency. Our previous testing assumed the `plugins` folder.